compiler/modules/front/codeGen.bal

```diff
@@ -777,11 +775,19 @@ function codeGenAssignStmt(CodeGenContext cx, bir:BasicBlock startBlock, Environ
     if lValue is s:VarRefExpr {
         return codeGenAssignToVar(cx, startBlock, env, lValue.varName, expr);
     }
+    else if lValue is s:WILDCARD {
+        return codeGenAssignToWildcard(cx, startBlock, env, expr);
+    }
     else {
         return codeGenAssignToMember(cx, startBlock, env, lValue, expr);
     }
 }
 
+function codeGenAssignToWildcard(CodeGenContext cx, bir:BasicBlock block, Environment env, s:Expr expr) returns CodeGenError|StmtEffect {
```

Comment:
This doesn't ned to be a separate function

Reply:
Removed